tests/nix_tests/test_ics20_precompile.py

```diff
@@ -1787,24 +1970,25 @@ def check_allowance_set():
 
     wait_for_fn("allowance has changed", check_allowance_set)
 
-    # send some funds (100 aevmos) to the contract to perform
+    # send some funds (1e18 aevmos) to the contract to perform
     # internal transfer within the tx
     src_bech32 = eth_to_bech32(src_acc.address)
     contract_bech32 = eth_to_bech32(eth_contract.address)
-    fund_tx = evmos.cosmos_cli().transfer(
+    scaling_factor = get_scaling_factor(cli)
+    fund_tx = cli.transfer(
         src_bech32,
         contract_bech32,
-        f"{initial_contract_balance}aevmos",
-        gas_prices=f"{evmos_gas_price + 100000}aevmos",
+        f"{initial_contract_balance}{evm_denom}",
+        gas_prices=f"{evmos_gas_price + 100000/scaling_factor:.8f}{evm_denom}",
```

Comment:
I don't get this here, why are we using the `.8f` conversion here? So basically the gas prices is going to be floating point value with 8 decimals? Why exactly 8 decimals? It's neither 6 nor 18, nor 18-6=12 so I'm confused where this number comes from 👀

Reply:
we can put 12 if you like it better. does not make big difference. Keep in mind that what really matters is the actual fees (gas_price * gas) that should be at least 0.000001 to avoid being 0 fees in the case of 6 dec precision. But you can still have a gas price that has more precision than 6 decimals